### PR TITLE
adding some data & pictures

### PR DESCRIPTION
**Pull request by jaalazard**
## Description

Please include a summary of the change or which issue is fixed.

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Area of change

- [ ] Frontend
- [x] Backend
- [ ] Other: .....

## General checklist:

- [x] My code follows the style guidelines of this project
- [x] I ran `bun run format`/`bun run lint` before commit
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation where needed
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] I have updated `develop` and merged to my branch before submitting pull request
- [x] My pull request generate no conflicts with `develop` branch
- [x] I requested code review from other team members

## Frontend checklist:

- [ ] I followed guidelines for `HTML`/`JSX`/`Typescript`
- [ ] My `Javascript` generate no new console errors
- [ ] I tested my code cross browsers
- [ ] My slice is pixel perfect for both desktop and mobile according to design
- [ ] I conducted basic QA to assure all features are working
- [ ] I tested responsive for mobile and tablet resolutions

## Backend checklist:

- [x] I tested admin by manually adding content from zero
- [x] I created easy to use admin experience which is self-explanatory
- [x] I added description to admin fields in hard-to-understand areas
- [x] I followed guidelines for naming `Typescript` variables
- [x] I conducted basic QA to assure all features are working

### CODE REVIEW

**Review by Cardamced — DISMISSED** [review]
Top.

**Review by No3ml3 — DISMISSED** [review]

**Review by Cardamced — DISMISSED** [review]
Yop !

File: package.json [hunk 1]
```diff
@@ -59,5 +59,8 @@
     "prettier": "^3.1.0",
     "prettier-plugin-tailwindcss": "^0.5.7",
     "typescript": "^5.3.2"
+  },
+  "dependencies": {
+    "react-refresh": "^0.14.0"
```
Comment (YDrogen) [inline package.json]: ?

**Review by YDrogen — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (jaalazard) [reply package.json]: je ne pouvais pas bun run dev sans ça :(

**Review by No3ml3 — APPROVED** [review]

**Review by Justicesvr — APPROVED** [review]

**Review by Cardamced — APPROVED** [review]
Yep !

**Review by YDrogen — APPROVED** [review]

**Pull request merged**